src/app/views/sidebar/resource-explorer/collection/APICollection.tsx

```diff
@@ -0,0 +1,287 @@
+} from '@fluentui/react';
+
+import { useAppDispatch, useAppSelector } from '../../../../../store';
+import { componentNames, eventTypes, telemetry } from '../../../../../telemetry';
+import { PopupsComponent } from '../../../../services/context/popups-context';
+import { usePopups } from '../../../../services/hooks';
+import { translateMessage } from '../../../../utils/translate-messages';
+import { downloadToLocal } from '../../../common/download';
+import Paths from './Paths';
+import { generatePostmanCollection, generateResourcePathsFromPostmanCollection } from './postman.util';
+import { addResourcePaths, removeResourcePaths } from '../../../../services/slices/collections.slice';
+import { useEffect, useState } from 'react';
+import { ResourcePath } from '../../../../../types/resources';
+import { setQueryResponseStatus } from '../../../../services/slices/query-status.slice';
+import { isGeneratedCollectionInCollection } from './upload-collection.util';
+
+export interface APICollection {
+  version: string;
+}
+
+const APICollection: React.FC<PopupsComponent<APICollection>> = (props) => {
+  const dispatch = useAppDispatch();
+  const { show: viewPermissions } = usePopups('collection-permissions', 'panel');
+  const { show: showPopup } = usePopups('edit-collection-panel', 'panel');
+  const { show: showEditScopePanel } = usePopups('edit-scope-panel', 'panel');
+  const { collections } = useAppSelector((state) => state.collections);
+  const [isDialogHidden, setIsDialogHidden] = useState(true);
+  const [uploadedCollections, setUploadedCollections] = useState<ResourcePath[]>([]);
+  const [fileInputKey, setFileInputKey] = useState(Date.now());
+  const[items, setItems] = useState<ResourcePath[]>([]);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    const timeoutId = setTimeout(() => {
+      if (collections && collections.length > 0) {
+        const defaultPaths = collections.find(k => k.isDefault)?.paths || [];
+        setItems(defaultPaths);
+        setLoading(false);
+      }
+    }, 300);
+
+    return () => clearTimeout(timeoutId);
+  }, [collections]);
+
+  if (loading) {
+    return <Label>{translateMessage('Loading collections...')}</Label>;
+  }
+
+  const columns = [
+    { key: 'url', name: 'URL', fieldName: 'url', minWidth: 300, maxWidth: 1100, isResizable: true },
+    { key: 'scope', name: 'Scope', fieldName: 'scope', minWidth: 150, maxWidth: 200, isResizable: true }
+  ];
+
+  const generateCollection = () => {
+    const content = generatePostmanCollection(items);
+    const filename = `${content.info.name}-${content.info._postman_id}.postman_collection.json`;
+    downloadToLocal(content, filename);
+    trackDownload(filename, componentNames.DOWNLOAD_POSTMAN_COLLECTION_BUTTON);
+  }
+
+  function trackDownload(filename: string, componentName: string) {
+    telemetry.trackEvent(eventTypes.BUTTON_CLICK_EVENT, {
+      componentName,
+      filename
+    });
+  }
```

Comment:
This is defined in `src/app/views/common/download.ts`. Choose one place to reduce duplication of code.